### PR TITLE
added one more sysout

### PR DESCRIPTION
**Pull request by sujanab9**

### CODE REVIEW

File: CrudOperations/src/main/java/com/code/crud/CrudOperationsApplication.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public void run(String... args) throws Exception {
           employeeRepository.save(employee2);
 
           System.out.println("save employees");
```
Comment (santhu538) [inline CrudOperations/src/main/java/com/code/crud/CrudOperationsApplication.java]: remove duplicate sysout

**Review by santhu538 — CHANGES_REQUESTED** [review]
Approved

File: CrudOperations/src/main/java/com/code/crud/CrudOperationsApplication.java (hunk 1 above)
Comment (sujanab9) [reply CrudOperations/src/main/java/com/code/crud/CrudOperationsApplication.java]: removed duplicate

**Review by santhu538 — APPROVED** [review]
Approved

**Pull request merged**